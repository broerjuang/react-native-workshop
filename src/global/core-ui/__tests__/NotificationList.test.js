// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import {NotificationList} from '../index';

describe('Icon', () => {
  it('should render Icon corectly', () => {
    let component: ReactTestRenderer = renderer.create(<NotificationList />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
