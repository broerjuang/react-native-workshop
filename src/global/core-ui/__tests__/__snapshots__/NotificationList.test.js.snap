// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render Icon corectly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "margin": 10,
      "shadowColor": "black",
      "shadowOffset": Object {
        "height": 1,
        "width": 1,
      },
      "shadowOpacity": 0.2,
    }
  }
/>
`;
